README.md

```diff
@@ -136,12 +137,11 @@ _(scroll right on table to see all 5 columns)_
-| Netflix                        | [github.com (screenshot)](https://github.com/karlhadwen/netflix/blob/master/netflix-preview.png)                                                                       | [GitHub](https://github.com/karlhadwen/netflix)                                                                                            | React, Firebase                                                                | ![GitHub Repo stars](https://flat.badgen.net/github/stars/karlhadwen/netflix)                                                                                                                    |
-| Netflix                        | [spaceflix.herokuapp](http://spaceflix.herokuapp.com/#/)                                                                                                               | [GitHub](https://github.com/ehecker/spaceflix)                                                                                             | Ruby, React, Redux, PostgreSQL, AWS                                            | ![GitHub Repo stars](https://flat.badgen.net/github/stars/ehecker/spaceflix)                                                                                                                     |
+| Netflix                        | [azazel5.github.io](https://azazel5.github.io/NetflixClone/) <br/> [github.com (gif)](https://github.com/Azazel5/NetflixClone/blob/master/flixdemo.gif)                | [GitHub](https://github.com/Azazel5/NetflixClone)                                                                                          | React, Redux                                                                   | ![GitHub Repo stars](https://flat.badgen.net/github/stars/Azazel5/NetflixClone)                                                                                                                  |
+| Netflix                        | [netflix-clone-dd230.web.app](https://netflix-clone-dd230.web.app/)[github.com (gif)](https://github.com/abodmicheal/React-projects/blob/master/gifs/20200915_142422.gif)                                                                                                    | [GitHub](https://github.com/abodmicheal/React-projects/tree/master/Netflix%20UI%20Web%20App)                                               | React, Firebase                                                                | ![GitHub Repo stars](https://flat.badgen.net/github/stars/abodmicheal/React-projects)                                                                                                            |                                                                                                                    |
```

Comment:
This project doesn't use Firebase except for hosting, Not sure if that should be included or not.
https://github.com/abodmicheal/React-projects/tree/master/Netflix%20UI%20Web%20App